Gemfile

```diff
@@ -1,12 +1,13 @@
 source "http://rubygems.org"
 
-ruby '2.1.6'
+# TODO: remove me
+# ruby '2.1.6'
```

Comment:
Hm, that made me realise that Nanoc 4 does not enforce any Ruby version in any way. Emitting a warning for older Ruby versions would probably be a good idea (or potentially even raising an error).

I *think* using Ruby 2.1 with Nanoc 4.0 will still work, but I know for a fact that 4.1 will require Ruby 2.2.